### PR TITLE
Translate contributing.md into Spanish

### PR DESCRIPTION
**Pull request by blaisep**
update the missing translation list.
@mariacamilagl perdona la demora y no tengas reparo en corregir. De pronto me di cuenta de que esta página representa la voz del jefe y presumo que @tiangolo querrá cambiarla a su gusto.

### CODE REVIEW

File: docs/es/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,520 @@
+# Desarollo - Aportando
```
Comment (AdrianDeAnda) [inline docs/es/docs/contributing.md:1]:
```suggestion
# Desarrollo - Aportando
```

File: docs/es/docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+
+Si te interesa formar parte del proyecto, empezar por revisar las maneras de [apoyar al proyecto FastAPI y de conseguir apoyo.](help-fastapi.md){.internal-link target=_blank}.
+
+## Desarollo
```
Comment (AdrianDeAnda) [inline docs/es/docs/contributing.md:5]:
```suggestion
## Desarrollo
```

File: docs/es/docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+
+---> 100%
+```
+
+</div>
+
+Vuelve a activar el entorno para asegurar que estás utilizando ese`flit` recién instalado,
+ y no algún otro del entorno global.
+
+Y ahora utilzas `flit` para installar las dependencias para el desarollo:
```
Comment (AdrianDeAnda) [inline docs/es/docs/contributing.md:119]:
```suggestion
Y ahora utilzas `flit` para installar las dependencias para el desarrollo:
```

File: docs/es/docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,520 @@
...
+La mayor parte de las pruebas se llevan a cabo contra los ejemplos de código en los documentos.
+
+
+De esta forma se compreuba que:
+
+* Los documentos están al día.
+* Los ejemplos se pueden ejectuar tál como están.
+* La mayoria de las prestaciones se describen en los documentos, comprobado por cobertura.
+
+Durante el desarollo local, se utiliza un script que construye el sitio y detecta cambios, recargando sobre la marcha.
```
Comment (AdrianDeAnda) [inline docs/es/docs/contributing.md:222]:
```suggestion
Durante el desarrollo local, se utiliza un script que construye el sitio y detecta cambios, recargando sobre la marcha.
```

Comment (blaisep) [reply docs/es/docs/contributing.md:222]: Oops!! Thank you AdrianDeAnda.

File: docs/es/docs/contributing.md (hunk 2 above)
Comment (blaisep) [reply docs/es/docs/contributing.md]: Muchisimas gracias

File: docs/es/docs/contributing.md [hunk 5]
```diff
@@ -0,0 +1,520 @@
...
+
+* Crea un sólo pull request por página traducida, para facilitar las revisiones y reseñas.
+
+Para aquellos idiomas que no puedo hablar, esperaré a que otros puedan revisar la traducción antes de incorporar.
+
+* También puedes investigar si hay traducciones que puedas reseñar, asi me ayudarás a saber si la traducción está lista para incorporar.
+
+* Sólo utiliza los ejemplos en los documentos y traduce el texto. No hace falta crear nada nuevo para que esto funcione.
+
+* Utiliz las mismas imágenes, nombres de ficheros y enlaces. No hace falta cambiar nada para que funcione.
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:299]:
```suggestion
* Utiliza las mismas imágenes, nombres de ficheros y enlaces. No hace falta cambiar nada para que funcione.
```

File: docs/es/docs/contributing.md [hunk 6]
```diff
@@ -0,0 +1,520 @@
...
+
+Ahora, supongamos que quieras añadir una traducción para la sección [Features](features.md){.internal-link target=_blank}.
+
+* Copia el fichero de:
+
+```
+docs/en/docs/features.md
+```
+
+* Pega el ficher en el mismo lugar, correspondiente al idioma al que quieres traducir, por ejemplo:
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:340]:
```suggestion
* Pega el fichero en el mismo lugar, correspondiente al idioma al que quieres traducir, por ejemplo:
```

File: docs/es/docs/contributing.md [hunk 7]
```diff
@@ -0,0 +1,520 @@
...
+
+Ahora podrás traducir el resto ye verás cómo aparece segun vayas progresando.
+
+#### Un idioma nuevo: partiendo de zero
+
+Digamos que quieras aportar traducciones para un idioma aún no traducido en ninguna de las páginas.
+
+Supongamos que quieres aportart traducciones al criollo haitiano, y todavía no aparece en los documentos.
+
+Consultando la tabla de códigos, veras que la secuencia para "Creole" es `ht`.
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:399]:
```suggestion
Consultando la tabla de códigos, verás que la secuencia para "Creole" es `ht`.
```

File: docs/es/docs/contributing.md [hunk 8]
```diff
@@ -0,0 +1,520 @@
...
+
+Lo sigiuente sería ejecutar el script que genera un nuevo directorio de traducciones:
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:401]:
```suggestion
Lo siguiente sería ejecutar el script que genera un nuevo directorio de traducciones:
```

File: docs/es/docs/contributing.md [hunk 9]
```diff
@@ -0,0 +1,520 @@
...
+$ python ./scripts/docs.py new-lang ht
+
+Successfully initialized: docs/ht
+Updating ht
+Updating en
+```
+
+</div>
+
+Now you can check in your code editor the newly created directory `docs/ht/`.
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:416]:
```suggestion
```

File: docs/es/docs/contributing.md [hunk 10]
```diff
@@ -0,0 +1,520 @@
...
+Ahora podrás ver el nuevo directorio  `docs/ht/` en tu editor.
+
+!!! tip
+    Crea el primer pull request con tan solo esto, incluso antes de añadir traducciones, para preparar la configuración del conjunto para el nuevo idioma.
+
+    Asi otros podrán aportar cambios mientras traduces. 🚀
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:422]:
```suggestion
    Así otros podrán aportar cambios mientras traduces. 🚀
```

File: docs/es/docs/contributing.md [hunk 11]
```diff
@@ -0,0 +1,520 @@
...
+
+```
+ raise TemplateNotFound(template)
+jinja2.exceptions.TemplateNotFound: partials/language/xx.html
+```
+
+quiere decir que el tema no apoya ese idioma (en este caso de debe al código inválido `xx`).
+
+No te apures, puedes cambiar el idioma del tema a inglés y proceder a traducir el contenido de los documentos.
+En ese case, edita el `mkdocs.yml` para tu nuevo idioma:
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:440]:
```suggestion
En ese caso, edita el `mkdocs.yml` para tu nuevo idioma:
```

File: docs/es/docs/contributing.md [hunk 12]
```diff
@@ -0,0 +1,520 @@
...
+  language: xx
+```
+
+Cambia el código del `xx` al código de `en`.
+
+Y vuelve a arrancar el servidor local.
+
+#### Prever los resultados
+
+Cuando ejectas el script `./scripts/docs.py` con el parámetro `live` sólo muestra los ficheros disponibles en el entorno local.
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:456]:
```suggestion
Cuando ejecutas el script `./scripts/docs.py` con el parámetro `live` sólo muestra los ficheros disponibles en el entorno local.
```

File: docs/es/docs/contributing.md [hunk 13]
```diff
@@ -0,0 +1,520 @@
...
+Successfully built docs for: es
+Copying en index.md to README.md
+```
+
+</div>
+
+Ahora tienes el conjunto entero de todos los documentos bajo `./docs_build/` en todos los idiomas. Se incluyen también aquellos ficheros que carecen de traducción, con
+una nota diciendo "this file doesn't have a translation yet". Ese directorio lo puedes ignorar.
+
+Luego, el comando construye el sitio de MkDocs para cada idioma, los combina y finalmente produce las salida dentro de `./site/`.
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:479]:
```suggestion
Luego, el comando construye el sitio de MkDocs para cada idioma, los combina y finalmente produce las salidas dentro de `./site/`.
```

File: docs/es/docs/contributing.md [hunk 14]
```diff
@@ -0,0 +1,520 @@
...
+  - en: /
+  - es: /es/
+- features.md
+```
+
+Si existieran otras referencias, asegurate de conservar la misma secuencia que en la versión de inglés.
+
+Si consultas tu buscador, verás que los documentos incluyen tu nueva sección. 🎉
+
+Ahora podrás traducir el resto ye verás cómo aparece segun vayas progresando.
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:391]:
```suggestion
Ahora podrás traducir el resto, ya verás cómo aparece según vayas progresando.
```

File: docs/es/docs/contributing.md [hunk 15]
```diff
@@ -0,0 +1,520 @@
...
+$ uvicorn tutorial001:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+como Uvicorn utiliza el puerto `8000` por defecto, los documentos en `8008` no confligen.
+
+### Translations
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:274]:
```suggestion
### Traducciones
```

File: docs/es/docs/contributing.md [hunk 16]
```diff
@@ -0,0 +1,520 @@
...
+
+Se agradecen muchísimo cualquier ayuda con los documententos, puesto que es imposible sin el apoyo de la comunidad. 🌎 🚀
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:276]:
```suggestion
Se agradece muchísimo cualquier ayuda con las traducciones, puesto que es imposible hacerlas sin el apoyo de la comunidad. 🌎 🚀
```

File: docs/es/docs/contributing.md [hunk 17]
```diff
@@ -0,0 +1,520 @@
...
+Sirve los documentos desde `http://127.0.0.1:8008`.
+Así podras editar documentos y código fuente y ver los resultados en vivo.
+
+#### Typer CLI (optional)
+
+Las siguentes instrucciones muestran cómo utilizar `./scripts/docs.py` con el programa `python` directamente.
+
+Tambien puedes aprovechar <a href="https://typer.tiangolo.com/typer-cli/" class="external-link" target="_blank">Typer CLI</a>, para obtener auto-completamiento ¿? .
+
+Si instalas Typer CLI, puedes obtener auto-completamiento:
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:245]:
```suggestion
Tambien puedes aprovechar <a href="https://typer.tiangolo.com/typer-cli/" class="external-link" target="_blank">Typer CLI</a>, para obtener auto-completado.

Si instalas Typer CLI, puedes obtener auto-completado:
```

File: docs/es/docs/contributing.md [hunk 18]
```diff
@@ -0,0 +1,520 @@
...
+<span style="color: green;">[INFO]</span> Start watching changes
+<span style="color: green;">[INFO]</span> Start detecting changes
+```
+
+</div>
+
+Sirve los documentos desde `http://127.0.0.1:8008`.
+Así podras editar documentos y código fuente y ver los resultados en vivo.
+
+#### Typer CLI (optional)
```
Comment (mariacamilagl) [inline docs/es/docs/contributing.md:239]:
```suggestion
#### Typer CLI (opcional)
```

**Review by mariacamilagl — CHANGES_REQUESTED** [review]
Thank you @blaisep for your contribution, and thank you @AdrianDeAnda for your review! 🎉 ☕ 

Also thanks for your patience! 😃 

I have added some suggestions that you can accept directly.

**Comment by blaisep** [comment]
Listo, @mariacamilagl

**Review by AdrianDeAnda — APPROVED** [review]

File: docs/es/docs/contributing.md [hunk 19]
```diff
@@ -0,0 +1,520 @@
...
+```
+
+</div>
+
+Puesto que ejecuta comandos en secuencia y también modifica muchos ficheros, toma más tiempo. Puede resultar más fácil utilizar `scripts/format.sh`
+frecuentemente y solamente `scripts/format-imports.sh` antes de someter.
+
+## Docs
+
+Lo primero es comprobar que tienes tu entorno tal como hemos descrito, as se instalarán todos los requisitos.
```
Comment (pedabraham) [inline docs/es/docs/contributing.md:193]:
```suggestion
Lo primero es comprobar que tienes tu entorno tal como hemos descrito, así se instalarán todos los requisitos.
```

File: docs/es/docs/contributing.md [hunk 20]
```diff
@@ -0,0 +1,520 @@
...
+
+Los documentos se desarollan con <a href="https://www.mkdocs.org/" class="external-link" target="_blank">MkDocs</a>.
+También se encuentran utlidades para facilitar las traducciones en `./scripts/docs.py`.
+
+!!! tip
+    No hace falta aceder al código en `./scripts/docs.py`, simplemente lo utilizas desde la linea de comandos.
+
+Todo el contenido de los documentos se encuentra en formato Markdown dentro del directorio `./docs/en/`. El texto en castellano se encuentra
```
Comment (pedabraham) [inline docs/es/docs/contributing.md:201]:
```suggestion
Todo el contenido de la documentación se encuentra en formato Markdown dentro del directorio `./docs/en/`. El texto en castellano se encuentra
```

File: docs/es/docs/contributing.md [hunk 21]
```diff
@@ -0,0 +1,520 @@
...
+### Docs for tests
+
+La mayor parte de las pruebas se llevan a cabo contra los ejemplos de código en los documentos.
+
+
+De esta forma se compreuba que:
+
+* Los documentos están al día.
+* Los ejemplos se pueden ejectuar tál como están.
+* La mayoria de las prestaciones se describen en los documentos, comprobado por cobertura.
```
Comment (pedabraham) [inline docs/es/docs/contributing.md:220]:
```suggestion
* La mayoría de las funciones se describen en los documentos, aseguradas por pruebas de cobertura.
```

**Review by pedabraham — CHANGES_REQUESTED** [review]

File: docs/es/docs/contributing.md [hunk 22]
```diff
@@ -0,0 +1,520 @@
+# Desarrollo - Contribuciones
+
+Si te interesa formar parte del proyecto, empieza por revisar las maneras de [apoyar a FastAPI y conseguir apoyo.](help-fastapi.md){.internal-link target=_blank}.
+
+## Desarrollo
+
+Si ya has copiado el repositorio y sabes con certeza que necesitas indagar a fondo el código fuente,
```
Comment (jovicon) [inline docs/es/docs/contributing.md:7]:
```suggestion
Si ya has copiado el repositorio y sabes con certeza que necesitas indagar a fondo del código fuente,
```

File: docs/es/docs/contributing.md [hunk 23]
```diff
@@ -0,0 +1,520 @@
...
+
+### Docs for tests
+
+La mayor parte de las pruebas se llevan a cabo contra los ejemplos de código en los documentos.
+
+
+De esta forma se compreuba que:
+
+* Los documentos están al día.
+* Los ejemplos se pueden ejectuar tál como están.
```
Comment (jovicon) [inline docs/es/docs/contributing.md:219]:
```suggestion
* Los ejemplos se pueden ejecutar tál como están.
```

File: docs/es/docs/contributing.md [hunk 24]
```diff
@@ -0,0 +1,520 @@
...
+
+```console
+$ uvicorn tutorial001:app --reload
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+como Uvicorn utiliza el puerto `8000` por defecto, los documentos en `8008` no confligen.
```
Comment (jovicon) [inline docs/es/docs/contributing.md:272]:
```suggestion
como Uvicorn utiliza el puerto `8000` por defecto, los documentos en el puerto `8008` no chocarán.
```

**Review by jovicon — CHANGES_REQUESTED** [review]
There are some proposed changes and some fixes too.

**Comment by blaisep** [comment]
Thank you @pedabraham , @jovicon , I think I committed all the fixes. I'm sorry for the delay. I moved from California to Massachusetts and I expect that now I will be more responsive.

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**